test/pdo_sqlsrv/pdo_prepare_options.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+Test PDO::prepare by passing in attributes
+--SKIPIF--
+
+--FILE--
+<?php
+  
+require_once("autonomous_setup.php");
+
+try 
+{   
+
+    class CustomPDOStatement extends PDOStatement
+    {
+        protected function __construct() {
+        }
+    }
+
+    $database = "tempdb";
+	$dsn = 	"sqlsrv:Server = $serverName; Database = $database";
+    $prep_attr = array(PDO::SQLSRV_ATTR_ENCODING => PDO::SQLSRV_ENCODING_UTF8,
+                       PDO::ATTR_STATEMENT_CLASS => array('CustomPDOStatement', array()),
+                       PDO::SQLSRV_ATTR_DIRECT_QUERY => true,
+                       PDO::ATTR_EMULATE_PREPARES => false,
+                       PDO::SQLSRV_ATTR_FETCHES_NUMERIC_TYPE => true
+                       );
+    $conn = new PDO( $dsn, $username, $password); 
```

Comment:
the other one is get_set_attribute. I think this is the only test that actually tests for valid options in a prepare statement